### DISCUSSION TITLE
Error introducing @wordpress/ Components in WordPress

### DISCUSSION THREAD

**Discussion by Bruse-Lee** [WordPress/gutenberg#42804] [Q&A]
I was about to create a block with a date picker using Gutenberg, and this error occurred in the background when I introduced the DateTimePicker component.

![image](https://user-images.githubusercontent.com/62770320/181717178-65aa85de-8cc7-4c23-9502-972e9b60deb2.png)

![image](https://user-images.githubusercontent.com/62770320/181717252-6122079c-9aaf-43ff-8978-ed2ef7e54ef6.png)

**Comment by talldan** [comment]
@Bruse-Lee What's the actual error? You'll need to click on the link in the error message in your console to view it. You can also share the link here (I don't feel like typing it out from an image 😄 )
